test/Microsoft.AspNetCore.JsonPatch.Test/JsonPatchDocumentJsonPropertyAttributeTest.cs

```diff
@@ -15,57 +15,66 @@ public class JsonPatchDocumentJsonPropertyAttributeTest
         [Fact]
         public void Add_ToRoot_OfListOfObjects_AtEndOfList()
         {
-            var patchDoc = new JsonPatchDocument<List<JsonPropertyObject>>();
-            patchDoc.Add(p => p, new JsonPropertyObject());
+            // Arrange
+            var patchDocument = new JsonPatchDocument<List<JsonPropertyObject>>();
+            patchDocument.Add(p => p, new JsonPropertyObject());
+
+            var serialized = JsonConvert.SerializeObject(patchDocument);
```

Comment:
>var serialized = JsonConvert.SerializeObject(patchDocument); [](start = 12, length = 60)

There are more cases for #5 in this file. You can probably do a find for `SerializeObject(patchDocument)` and find them all